### ISSUE TITLE
A merged Ticket is escalated

### ISSUE THREAD

**Issue by schnudd31do3** [znuny/Znuny#200]
### Environment

- Znuny version: 6.2.2

### Expected behaviour

A merged ticket is a kind of closed and therefore should not appear in the view of the escalated tickets.

### Actual behaviour

![image](https://user-images.githubusercontent.com/72391822/151578177-4ae01d8a-1419-4fce-b627-7b260158bf6e.png)

![image](https://user-images.githubusercontent.com/72391822/151578279-a2f2adcd-8193-4672-86a2-f48e61da872b.png)

![image](https://user-images.githubusercontent.com/72391822/151578319-8c0592f4-95ac-4104-b483-d63268cbc1d8.png)

Is it not allowed to merge a newer ticket when a older one should be continued?

**Comment by hanneshal** [comment]
Hi @schnudd31do3 
This looks kinda strange. Non of the tickets is actually escalated.  (Nothing is shown on the ticket details)

So the error in my opinion is more like "not escalated ticket is shown in the widget for escalated tickets". The ticket detail view of the merged ticket looks fine. 
This could be a cache issue. The widgets have their own ttl. Another one could be an error of the escalation index builder. 

Does your daemon work fine?
./otrs.Console Maint::Daemon::Summary 

Can you rebuild the escalation index manually, please.

Another thing is: merged is not a visible state by default. So merged tickets should not appear anywhere in the first place.

This could also be a cache issue or could be related to a wrong configuration.

Thx

**Comment by schnudd31do3** [comment]
Hello Johannes,

how to do „Can you rebuild the escalation index manually, please“

The „./otrs.Console Maint::Daemon::Summary“ reports no fail.

Perhape it could have something to do with the migration from 6.0 to 6.1/6.2?

Regards,

Volker

**Comment by hanneshal** [comment]
It can be related to many things. But a migration itself is usually not one of them. After a migration users usually find more things that are not as expected, because they are more cautious about it. 
Or manual modifications are removed and trigger "bugs".

Often simple configuration stuff, modified years ago, is the cause for this.
Anyway:
To clear the cache and rebuild the escalation index you can use the following commands:
otrs.Console.pl Maint::Cache::Delete
otrs.Console.pl Maint::Ticket::EscalationIndexRebuild

Maybe this helps already. 
Btw: I tested it with a regular installation and it works as expected. Merged tickets are not shown in overviews or widgets.

regards

**Comment by schnudd31do3** [comment]
I did the two steps:
otrs.Console.pl Maint::Cache::Delete
otrs.Console.pl Maint::Ticket::EscalationIndexRebuild

The merged ticket is still visible in the view of escalated tickets. I guess you are right, it could be in context with upgrading from 6.0->6.1->to 6.2. But is it possible to manipulate the ticket in a particular way that it will no more shown there?

I found also a closed ticket in the view of the escalated tickets:
![image](https://user-images.githubusercontent.com/72391822/151697164-cb490326-b4d2-4f31-b8ba-59e2674b5d40.png)
